corehq/apps/app_manager/templates/app_manager/form_view_base.html

```diff
@@ -103,9 +103,18 @@
             var isModuleFilterEnabled = {{ is_module_filter_enabled|BOOL }};
             var formFilter = {{ form.form_filter|JSON }};
             var allOtherFormsRequireACase = {{ form.all_other_forms_require_a_case|BOOL }};
+            var formFilterReferencesCase = {{ form_filter_references_case|BOOL }};
             var formFilterAllowed = ko.computed(function () {
```

Comment:
Previously, if module filtering was enabled, then form filtering was disabled, to prevent app builders from trying to use case properties that aren't available. We decided to revert this and leave the responsibility with app builders. The ticket was [FB 217791][1], and the PR was #10611. 

[menus.py:33][2] assumes the form filter is valid, so no change was needed there.

  [1]: http://manage.dimagi.com/default.asp?217791
  [2]: https://github.com/dimagi/commcare-hq/pull/10972/files#diff-1a371a01bdfc54712b051c92786a1682R33